### PR TITLE
Disguisting lavaland port

### PR DESCRIPTION
**Pull request by SkullyRoberts**
So, Spartan nagged me to do a PR for my lavaland port, without the waterworld stuff. I still need to move a crapton of the edits from waterworld onto here.
So, this is what I have right now. It's stable, it runs, and it is more or less just the lavaland map with the generation and all, but no ruins, actual lava, or any of the boss monsters. It uses the mobs from the asteroid.

### CODE REVIEW

File: _maps/cyberiad.dm [hunk 1]
```diff
@@ -20,17 +20,19 @@ z7 = empty
...
 DECLARE_LEVEL(TELECOMMS, CROSSLINKED, list(REACHABLE, BOOSTS_SIGNAL, AI_OK)),\
 DECLARE_LEVEL(CONSTRUCTION, CROSSLINKED, list(REACHABLE)),\
-DECLARE_LEVEL(MINING, CROSSLINKED, list(REACHABLE, STATION_CONTACT, HAS_WEATHER, ORE_LEVEL, AI_OK)),\
 DECLARE_LEVEL(DERELICT, CROSSLINKED, list(REACHABLE)),\
+DECLARE_LEVEL(LAVALAND, SELFLOOPING, list(REACHABLE, ORE_LEVEL)),\
 DECLARE_LEVEL(EMPTY_AREA, CROSSLINKED, list(REACHABLE)))
+//DECLARE_LEVEL(MINING, CROSSLINKED, list(REACHABLE, STATION_CONTACT, HAS_WEATHER, ORE_LEVEL, AI_OK)),\
 
-	#define USING_MAP_DATUM /datum/map/cyberiad
+	#define USING_MAP_DATUM /datum/map/
```
Comment (tigercat2000) [inline _maps/cyberiad.dm]: Why did you change this?

Comment (SkullyRoberts) [reply _maps/cyberiad.dm]: That's a good question. I don't remember doing that. XP

Comment (SkullyRoberts) [reply _maps/cyberiad.dm]: fixed it

**Comment by Squirgenheimer** [comment]
Merge commits from upstream are probably not optimal. Use rebase instead of merge. (Made a comment about this [here](https://github.com/ParadiseSS13/Paradise/pull/8746#comment-372030311))

I think squashing with interactive rebase may also help to reduce the number of commits if desired.

**Comment by Crazylemon64** [comment]
This is not really salvagable or implemented in a practical way, so this and #8769 are being closed.
